src/main/java/org/jitsi/jigasi/TranscriptionGatewaySession.java

```diff
@@ -228,100 +231,51 @@ void onJvbConferenceWillStop(JvbConference jvbConference, int reasonCode,
     }
 
     @Override
-    void notifyMemberJoined(ChatRoomMember chatMember)
+    void notifyChatRoomMemberJoined(ChatRoomMember chatMember)
     {
-        super.notifyMemberJoined(chatMember);
+        super.notifyChatRoomMemberJoined(chatMember);
 
-        ConferenceMember confMember = findMatchingConferenceMember(chatMember);
+        String identifier = getParticipantIdentifier(chatMember);
 
-        if (confMember == null)
-        {
-            logger.debug("Starting WaitThread for joining ChatRoomMember "+
-                    chatMember.getDisplayName());
-            new WaitForConferenceMemberThread(chatMember,
-                    this::addParticipant).start();
-        }
-        else
-        {
-            addParticipant(chatMember, confMember);
-        }
+        this.transcriber.maybeAddParticipant(identifier);
```

Comment:
Why do we need 3 function calls here? Use one function and let the `Transcriber` figure it out